media/js/base/fxa-utm-referral.js

```diff
@@ -14,6 +14,7 @@ if (typeof window.Mozilla === 'undefined') {
 
     var allowedList = [
         'https://accounts.firefox.com/',
+        'https://accounts.stage.mozaws.net/',
```

Comment:
Let's add `https://stable.dev.lcip.org/` and `https://accounts.firefox.com.cn/` here for parity with `mozilla-fxa-link.js`?